src/components/CharactersPage/CharactersTable/CharactersTable.js

```diff
@@ -86,11 +86,51 @@ const CharactersTable = ({ onRowClick }) => {
     },
   ]
 
+  const rows = data.allPeople.edges.map((edge) => edge.node)
+  const pageInfo = data.allPeople.pageInfo
+  const totalRowsCount = data.allPeople.totalCount
+  const slicedRows = rows.slice(
+    tablePage * rowsPerPage,
+    tablePage * rowsPerPage + rowsPerPage
+  )
```

Comment:
Any logic you have in a functional component that iterates through an array should be memoized for performance reasons